Lib/uuid.py

```diff
@@ -656,7 +703,12 @@ def _random_getnode():
 
 _node = None
 
-def getnode():
+_NODE_GETTERS_WIN32 = [_windll_getnode, _netbios_getnode, _ipconfig_getnode]
+
+_NODE_GETTERS_UNIX = [_unix_getnode, _ifconfig_getnode, _ip_getnode,
+                      _arp_getnode, _lanscan_getnode, _netstat_getnode]
+
+def getnode(*, getters=None):
```

Comment:
Can't see the purpose of any changes from here and further.

About this one specifically:
* There's no discussion about an interface change or justification for it.
* In any case, the `getters` arg is unused as the code is now.